pkg/operation/shoot/types.go

```diff
@@ -46,6 +46,7 @@ type Shoot struct {
 	OperatingSystemConfigsMap map[string]OperatingSystemConfigs
 	Extensions                map[string]Extension
 	InfrastructureStatus      []byte
+	InfrastructureNodesCIDR   *string
```

Comment:
Do you think this field is still required after you write back the node CIDR to the shoot?

Reply:
Great feedback, @timuthy! Indeed, it's no longer required now. Also, I removed the check validating the disjointedness of the networks as it's done by the API server when trying to write the nodes CIDR into the shoot spec. Great!